static/javascript/realtime.js

```diff
@@ -132,17 +139,40 @@ function createGraphs()
 						ticks: {
 							beginAtZero: true,
 							min: 0,
-							suggestedMax: 10
+							max: 10
```

Comment:
Why not go ahead and set the min and max values here since you got them above, instead of having a separate function?